New eiscp commands added

nitaybz opened the pull request:
## Status
READY

## Description
Added new file with variuos new acceptable eiscp commands

jupe commented on lib/Onkyo.js:
```diff
@@ -27,6 +27,7 @@ const Promise = require('bluebird');
 const defautLogger = require('./logger');
 const OnkyoError = require('./OnkyoError');
 const OnkyoCmds = require('./onkyo.commands.js');
```
could we drop old list ? doesn’t new file contains all these commands as well? probably needs some refactoring but I would not like to keep two files for same purpose..

nitaybz replied:
It will require a complete refactoring as your entire code is built on the old format while the new file is used for validation mostly.

jupe replied:
that's true. Maybe it's still worth of work overall. It's harder to keep backward compatibility but I think it's totally accepted to break everything as long as we increase major number.. I think this PR is okay approach for now as it brings more supported commands..  In future I would like to refactor library so that we doesn't need duplicate files but sure it needs some efforts..

jupe reviewed (commented):
could you add some basic unit tests for new functionality, otherwise I'm ready for merging.

nitaybz commented:
Sure I'll add it later today

nitaybz commented:
- did some fixing to the new commands payload
- added new unknown command "NDN"
- added 2 tests: 1 for commands from the eiscp commands file, and 1 unknown command

nitaybz commented:
Thanks... can you push this to npm please?
